src/common/guards/jwt-guard/jwt.guard.ts

```diff
@@ -1,9 +1,25 @@
-import { Injectable } from '@nestjs/common';
+import { ExecutionContext, Injectable } from '@nestjs/common';
+import { Reflector } from '@nestjs/core';
 import { AuthGuard } from '@nestjs/passport';
+import { Observable } from 'rxjs';
 
 @Injectable()
 export class JwtGuard extends AuthGuard('jwt') {
-  constructor() {
+  constructor(private reflector: Reflector) {
     super();
   }
+
+  canActivate(
```

Comment:
let's not use public guard. Rather turn off authorization globally, and then add @UseGuards(SomeGuard) in protected APIs